geo/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "geo"
 description = "Geospatial primitives and algorithms"
-version = "0.25.0"
+version = "0.26.0"
```

Comment:
Should this be 0.25.1 to match the changelog, or vice versa?

Reply:
Yep!